src/main/java/com/minecolonies/coremod/entity/pathfinding/ChunkCache.java

```diff
@@ -69,9 +67,9 @@ public ChunkCache(Level worldIn, BlockPos posFromIn, BlockPos posToIn, int subIn
         {
             for (int l = this.chunkZ; l <= j; ++l)
```

Comment:
Im not sure this wouldnt just break somewhere else. We should probably prevent this much earlier to reach this place in the first place then

Reply:
Right now this is just meant to prevent the classcast exception, I tested it and it still works.
It's up to Alex's Mobs to just not call it on the client.

Reply:
Did you test that it doesnt just now crash further down the road because of the missing chunkcache?

Reply:
Yep, I threw it on them, and dragged them out for multiple chunks far, nothing special happened, not even anything present in the log files that would indicate an issue.

After releasing them they just continued to path normally,